addons/package_hierarchy/models/stock_quant_package.py

```diff
@@ -246,3 +244,31 @@ def get_key(x):
             ):
                 return False
         return True
+
+    def _get_all_package_links(self):
```

Comment:
The name `_get_all_package_links` suggest to me that we are just retrieving links, but on line `260` we're creating a link if for the current package if it doesn't have any already (and potentially again on line `269`). I can't think of a better name though... Will see if I think of something in the morning.

Reply:
At the beginning was only searching, later Miquel suggested to create in case not found :)
Didnt change the name of method, tbh didnt even think to change it :)

Reply:
Let's leave as is for now, I can't think of an alternative.

Reply:
What about _get_package_link_structure() and add a docstring?